x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/exceptions/exceptions_table.tsx

```diff
@@ -0,0 +1,205 @@
+import { AllExceptionListsColumns, getAllExceptionListsColumns } from './columns';
+import { useAllExceptionLists } from './use_all_exception_lists';
+
+// Known lost battle with Eui :(
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const MyEuiBasicTable = styled(EuiBasicTable as any)`` as any;
+
+export type Func = () => void;
+export interface ExceptionListFilter {
+  name?: string | null;
+  list_id?: string | null;
+  created_by?: string | null;
+}
+
+interface ExceptionListsTableProps {
+  history: History;
+  hasNoPermissions: boolean;
+  loading: boolean;
+  formatUrl: FormatUrl;
+}
+
+export const ExceptionListsTable = React.memo<ExceptionListsTableProps>(
+  ({ formatUrl, history, hasNoPermissions, loading }) => {
+    const {
+      services: { http, notifications },
+    } = useKibana();
+    const [filters, setFilters] = useState<ExceptionListFilter>({
+      name: null,
+      list_id: null,
+      created_by: null,
+    });
+    const [loadingExceptions, exceptions, pagination, refreshExceptions] = useExceptionLists({
+      errorMessage: i18n.ERROR_EXCEPTION_LISTS,
+      filterOptions: filters,
+      http,
+      namespaceTypes: ['single', 'agnostic'],
+      notifications,
+      showTrustedApps: false,
+    });
+    const [loadingTableInfo, data] = useAllExceptionLists({
+      exceptionLists: exceptions ?? [],
+    });
+    const [initLoading, setInitLoading] = useState(true);
+    const [lastUpdated, setLastUpdated] = useState(Date.now());
+
+    const handleDelete = useCallback((id: string) => () => {}, []);
+
+    const handleExport = useCallback((id: string) => () => {}, []);
+
+    const exceptionsColumns = useMemo((): AllExceptionListsColumns[] => {
+      return getAllExceptionListsColumns(handleExport, handleDelete, history, formatUrl);
+    }, [handleExport, handleDelete, history, formatUrl]);
+
+    const handleRefresh = useCallback((): void => {
+      if (refreshExceptions != null) {
+        setLastUpdated(Date.now());
+        refreshExceptions();
+      }
+    }, [refreshExceptions]);
+
+    useEffect(() => {
+      if (initLoading && !loading && !loadingExceptions && !loadingTableInfo) {
+        setInitLoading(false);
+      }
+    }, [initLoading, loading, loadingExceptions, loadingTableInfo]);
+
+    const emptyPrompt = useMemo((): JSX.Element => {
+      return (
+        <EuiEmptyPrompt
+          title={<h3>{i18n.NO_EXCEPTION_LISTS}</h3>}
+          titleSize="xs"
+          body={i18n.NO_LISTS_BODY}
+        />
+      );
+    }, []);
+
+    const handleSearch = useCallback((search: string) => {
+      const regex = search.split(/\s+(?=([^"]*"[^"]*")*[^"]*$)/);
+      const formattedFilter = regex
+        .filter((c) => c != null)
+        .reduce<ExceptionListFilter>(
+          (filter, term) => {
+            const [qualifier, value] = term.split(':');
+
+            if (qualifier == null) {
+              filter.name = search;
```

Comment:
I don't think this is working as intended. Typing in a string into the search bar doesn't execute a search. It works for `list_id: <id>` but not for name searches.
![image](https://user-images.githubusercontent.com/616158/102246002-3ea23380-3ecc-11eb-975e-6f0160c6e829.png)

Reply:
Addressed in https://github.com/elastic/kibana/pull/86135